psalm-baseline.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
One last thing: we probably need to `export-ignore` this 😁

Reply:
Already there 💪 https://github.com/laminas/laminas-cli/blame/1.8.x/.gitattributes